src/main/java/com/alkemy/ong/dto/SlidesDto.java

```diff
@@ -0,0 +1,26 @@
+package com.alkemy.ong.dto;
+
+import com.alkemy.ong.model.Organization;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class SlidesDto {
```

Comment:
Estas devolviendo mas datos de lo que se solicitan, la tarjeta dice: _Devolverá el listado de Slides, con la imagen y el orden del mismo_, es decir, tu DTO deberia tener simplemente la imagen (imageUrl) y el orden (order). Al orden is queres ponele _position_ ya que en otro PR se modifico ese nombre porque order es una palabra reservada y trae ciertos conflictos.
El campo _text_ no se a que hace referencia, acabo de ver que tambien esta en la entidad, no recuerdo de que trata pero aca lo podes sacar sin problema